tests/plugins/test_resolve_composes.py

```diff
@@ -1118,6 +1118,55 @@ def test_only_pulp_repos(self, workflow):
             .and_return(ODCS_COMPOSE))
         self.run_plugin_with_args(workflow)
 
+    @pytest.mark.parametrize(('content_sets', 'build_only_content_sets'), (
+        (True, True),
+        (True, False),
+        (False, True),
+        (False, False),
+    ))
+    def test_only_content_sets(self, workflow, content_sets, build_only_content_sets):
+        main_cs_list = ['pulp-spam-rpms', 'pulp-bacon-rpms', 'pulp-eggs-rpms']
+        build_only_cs_list = ['dev-spam-rpms', 'dev-bacon-rpms', 'dev-eggs-rpms', 'pulp-spam-rpms']
+
+        if content_sets:
+            cs_json = {'x86_64': main_cs_list}
+            workflow._tmpdir.join('content_sets.yml').write(yaml.safe_dump(cs_json))
+
+        containter_json = {'compose': {'pulp_repos': True}}
```

Comment:
```suggestion
        container_json = {'compose': {'pulp_repos': True}}
```

and also a couple of times below here...

Reply:
fixed